Forms/MainForm.cs

```diff
@@ -801,6 +856,94 @@ private void ToggleCheckBoxes()
             }
         }
 
+        public void ApplyPreset(Presets set)
+        {
+
+            tbPreset.Text = "";
+            if (set == Presets.Default) //Default
+            {
+                tSString.Text = "zwetf--16psr--";
```

Comment:
These will be very annoying to have to maintain. Can we remove the built-in presets and just support sharing of config files instead?